SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraph.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "ReplicatedTestActorBase.h"
+#include "ReplicatedTestActorBase_RepGraph.generated.h"
+
+/**
+ * A replicated Actor which is always replicated by RepGraph regardless of client view.
+ */
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API AReplicatedTestActorBase_RepGraph : public AReplicatedTestActorBase
```

Comment:
You're gonna hate me, but maybe this actor should be named something else.
AReplicatedTestActorBase_RepGraphAlwaysReplicate ?